lib/elixir_analyzer/test_suite/captains_log.ex

```diff
@@ -31,6 +31,7 @@ defmodule ElixirAnalyzer.TestSuite.CaptainsLog do
     calling_fn module: CaptainsLog, name: :random_stardate
     called_fn module: :rand, name: :uniform
     comment Constants.captains_log_use_rand_uniform()
+    suppress_if Constants.solution_deprecated_random_module()
```

Comment:
@jiegillet I'm trying to suppress an exercise-specific check if a common check fails, but it doesn't work so far. Is it supposed to work?

Reply:
Try this

```suggestion
    suppress_if Constants.solution_deprecated_random_module(), :fail
```

Reply:
Ah! Perfect.

For some reason I thought it needs to be a tuple, so I was attempting:
```elixir
suppress_if {Constants.solution_deprecated_random_module(), :fail}
```

which didn't work but also didn't give any hints about why

Reply:
Ah, maybe we should add an error message then, it wouldn't be too hard to do.